### DISCUSSION TITLE
Changing computers - What to transfer and where?

### DISCUSSION THREAD

**Discussion by Sybille-kil** [GetPublii/Publii#1708] [Help]
Hello, I kept an old version of Publii for a long time because my Mac was too old and I could not update the system and therefore the new versions of Publii did not work properly on this old system. I have finally acquired a new Mac and I would like to know how to transfer the content of my Publii blog from one Mac to another, without risking losing content. My Mac runs Sequoia 15.0 Thank you.

**Comment by dziudek** [comment]
@Sybille-kil - most probably the easiest way will be to copy your /users/X/Documents/Publii catalog to a new computer. If the username is the same on both computers then it should be all. 

if the username is different - open /users/X/documents/Publii/config/app-config.json file and update "sitesLocation" field to a correct path.

I assume that you have sites and backups located under the /suers/X/Documents/Publii catalog - in other cases you must to copy these folders between computers and update paths manually.

Also if you are not migrating your account to a new computer, most probably you will need to set up server passwords again as they are stored in Keychain

**Reply by Sybille-kil** [reply]
Thank you very much for your answer, I will try your solutions. Cheers :)

**Reply by IO101010** [reply]
> Thank you very much for your answer, I will try your solutions. Cheers :)

Did it work? I will have the same question soon :)
